### PR TITLE
feat(API,CLI): add delete stack functionality

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: cmd/rokka/cli/stacks.go [hunk 1]
```diff
@@ -270,6 +270,10 @@ func listStacks(c *rokka.Client, args []string) (interface{}, error) {
 	return c.ListStacks(args[0])
 }
 
+func deleteStack(c *rokka.Client, args []string) (interface{}, error) {
+	return "", c.DeleteStack(args[0], args[1])
```
Comment (dpacassi) [inline cmd/rokka/cli/stacks.go]: Hmm. Do we have a particular reason why we don't return
`nil, c.DeleteStack(args[0], args[1])`?
I've seen this a multiple times so far and maybe we need to correct them (or correct me).

An empty string for me is still content, just empty content. But context wise, nil would be more suitable for this case?

**Review by dpacassi — COMMENTED** [review]
Looks good so far, there's just one point of discussion

File: cmd/rokka/cli/stacks.go (hunk 1 above)
Comment (mweibel) [reply cmd/rokka/cli/stacks.go]: you're right. will fix

**Comment by mweibel** [comment]
Fixed, thanks @dpacassi

**Pull request merged**